packages/microeditor-envelope/src/__tests__/GwtEditorWrapper.test.ts

```diff
@@ -0,0 +1,33 @@
+import { GwtEditor } from "../gwt/GwtEditor";
+import { GwtEditorWrapper } from "../gwt/GwtEditorWrapper";
+import { DummyEditor } from "./DummyEditor";
+
+const MockEditor = jest.fn(() => ({
+  getContent: jest.fn(),
+  setContent: jest.fn(),
+  isDirty: jest.fn()
+}));
+
+const mockEditor = new MockEditor();
+const wrapper = new GwtEditorWrapper(mockEditor);
+
+describe("GwtEditorWrapper", () => {
+  test("set content", () => {
+    wrapper.setContent(" a content ");
+    expect(mockEditor.setContent).toHaveBeenCalledWith("a content");
+  });
+  test("af_onOpen removes header", () => {
+    const parent = document.createElement("div");
+    const workbenchHeaderPanel = document.createElement("div");
+    const listBarHeading = document.createElement("div");
+    listBarHeading.className = ".panel-heading.uf-listbar-panel-header";
+    workbenchHeaderPanel.id = "workbenchHeaderPanel";
+    parent.appendChild(workbenchHeaderPanel);
+    document.body.appendChild(parent);    
+    wrapper.af_onOpen();
+    const removedHeaderPanel = document.getElementById("workbenchHeaderPanel");
+    const removedListBarHeader = document.querySelector(".panel-heading.uf-listbar-panel-header");
+    expect(removedHeaderPanel).toBeFalsy();
+    expect(removedListBarHeader).toBeFalsy();
+  });
```

Comment:
I think you could add a test that checks for a no-op behaviour when these elements don't exist as well.. It's a good thing to know that a method that's not supposed to fail, doesn't...

Reply:
Also, on this test, I think it's a good thing to improve readability to vertically space things apart. To make it clearer I would wrap `wrapper.af_onOpen()` aroud blank lines so that it is clear what is the test preparation, what is the method we're testing and what are the assertions.